hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/security/acl/TestParentAcl.java

```diff
@@ -148,11 +148,11 @@ public void testKeyAcl()
     List<OzoneAcl> originalBuckAcls = getBucketAcls(vol, buck);
     List<OzoneAcl> originalKeyAcls = getBucketAcls(vol, buck);
 
-    testParentChild(keyObj, WRITE, WRITE_ACL);
+    testParentChild(keyObj, READ, WRITE_ACL);
```

Comment:
@sumitagrawl , can you remove the rule comments on the top of this function?

Reply:
@ChenSammi Updated